qa/L0_storage_azure/test.sh

```diff
@@ -106,28 +106,22 @@ sleep 10
 
 # Test 1 Scenarios:
 # 1. access blob using shared key in envs
-# 2. adding more scenarios in future 
+# 2. adding more scenarios in future
 for ENV_VAR in "shared_key"; do
     SERVER_LOG=$SERVER_LOG_BASE.$ENV_VAR.log
     CLIENT_LOG=$CLIENT_LOG_BASE.$ENV_VAR.log
     MODEL_REPO="${AS_URL}/models"
-    if [ "$ENV_VAR" == "sas" ]; then
-        unset AZURE_STORAGE_KEY
-        sas=`az storage blob generate-sas --container-name ${CONTAINER_NAME} --account-name ${ACCOUNT_NAME} --account-key ${ACCOUNT_KEY} --name models`
-        sas_without_quote=$(eval echo $sas)
-        export AZURE_STORAGE_SAS="?$sas_without_quote"
-    fi
```

Comment:
@CoderHam can this be removed?

Reply:
We want to keep this for the sas scenario we will add in the future.

Reply:
@bnookala please keep this section in the test script